modules/upgrade/pages/db-migration.adoc

```diff
@@ -1,61 +1,65 @@
 [[db-migration]]
 = Database Migration
 
+This is about PostgreSQL database migration.
 
-When you installed {productname} Server 3.1 or earlier on {sls}{nbsp}12{nbsp}SP2 or earlier, it was based on a PostgreSQL{nbsp}9.4 database.
-PostgreSQL{nbsp}9.4 can be upgraded to PostgreSQL{nbsp}9.6.
+If you want to upgrade {productname} 3.2 to {productname} 4 you must check whether {productname} 3.2 is already based PostgreSQL version 10.
+If yes, you can skip this section.
+If it is based on a older one such as version 9.6 you must migrate PostgreSQL to version 10 first.
+For more information about migrating PostgreSQL 9.6 to version 10, see xref:db-migration.adoc#db-migration-prepare[].
+If you still run PostgreSQL 9.4 on {productname} 3.2 see the 3.2 product documentation at https://www.suse.com/documentation/suse-manager-3/3.2/susemanager-best-practices/html/book.suma.best.practices/bp.sp.migration.html#sp.migration.postgresql.
```

Comment:
```suggestion
If you run PostgreSQL{nbsp}9.4 on {productname} 3.2, see the 3.2 product documentation: https://www.suse.com/documentation/suse-manager-3/3.2/susemanager-best-practices/html/book.suma.best.practices/bp.sp.migration.html#sp.migration.postgresql.
```

Reply:
I wonder whether we should mention 9.4 at all in the first place. Quite some time ago we introduced some change that requires a "new" feature of postgresql that was not available in 9.4. So if somebody is still running this ancient postgresql version, he definitely also is running an totally outdated version of SUSE Manager. I think 9.4 is unusable for SUSE Manager since longer than a year.

Reply:
I see but I nevertheless would like to mention this case.  Of course, if developers or supporter would think this will cause more confusion than help, I can remove it.

Reply:
Honestly I do not care. IMHO it is just a no-op by now. I bet nobody is using 9.4 any longer.